Airport/Program.cs

```diff
@@ -0,0 +1,577 @@
+        public static int id = 0;
+        //Deserialize Flight
+        readonly static string filenameFlights = $"{jsonPath}flights.json";
+        readonly static string jsonTextFlights = File.ReadAllText(filenameFlights);
+        readonly static List<Flights> flights = JsonSerializer.Deserialize<List<Flights>>(jsonTextFlights);
+        //Deserialize Passenger
+        readonly static string filePassengers = $"{jsonPath}passenger.json";
+        readonly static string jsonTextPass = File.ReadAllText(filePassengers);
+        readonly static List<Passengers> passengers = JsonSerializer.Deserialize<List<Passengers>>(jsonTextPass);
+        //Deserialize AirCarrier
+        readonly static string filename1 = $"{jsonPath}airCarriers.json";
+        readonly static string jsonTextAir = File.ReadAllText(filename1);
+        readonly static List<AirCarrier> airCarriers = JsonSerializer.Deserialize<List<AirCarrier>>(jsonTextAir);
+
+
+
+        static void Main(string[] args)
+        {
+            try
+            {
+                InitialMenu();
+            }
+            catch (IndexOutOfRangeException ex)
+            {
+                ExceptionMethod(ex);
+            }
+            catch (FormatException ex)
+            {
+                ExceptionMethod(ex);
+            }
+        }
+
+        public static void ExceptionMethod(Exception e)
+        {
+            Console.WriteLine(e.Message);
+            Console.WriteLine(e.StackTrace);
+        }
+
+        public static void InitialMenu()
+        {
+            bool done = default;
+            while (!done)
+            {
+                Console.Clear();
+                WriteLogo();
+                MenuItems(1, "Login");
+                MenuItems(2, "Check buy a ticket");
+                MenuItems(3, "Check booking status");
+                MenuItems(4, "Logout");
+                var optionInit = int.Parse(Console.ReadLine());
+                switch (optionInit)
+                {
+                    case 1:
+                        Login();
+                        break;
+                    case 2:
+                        Buy();
+                        break;
+                    case 3:
+                        CheckBookingStatus();
+                        break;
+                    case 4:
+                        done = true;
+                        break;
+                    default:
+                        Console.WriteLine("Enter valid option!");
+                        Thread.Sleep(1000);
+                        break;
+                }
+            }
+        }
+
+        public static void MainMenu()
+        {
+            Console.Clear();
+            foreach (var item in airCarriers)
+            {
+                if (id == item.Id)
+                {
+                    Console.WriteLine(item.Name);
+                }
+            }
+            MenuItems(1, "Flights");
```

Comment:
you can store your menu items as dictionary, so no need for switch case that is below, like
`var dict = new Dictionary<string, Action>{{1, ShowAllFlights}}`
and instead of switch case it would be like `dict[option].Value.Invoke()`

in some cases instead of `Action` can be used `Func`

https://www.tutorialsteacher.com/csharp/csharp-action-delegate
https://www.tutorialsteacher.com/csharp/csharp-func-delegate